src/Polly.Extensions/Telemetry/Log.cs

```diff
@@ -61,12 +68,12 @@ internal static partial class Log
     [LoggerMessage(
         EventId = 3,
         Message = "Execution attempt. " +
-                "Source: '{PipelineName}/{PipelineInstance}/{StrategyName}', " +
-                "Operation Key: '{OperationKey}', " +
-                "Result: '{Result}', " +
+                SourceWithStrategy + Separator +
+                OperationKey + Separator +
+                Result + Separator +
                 "Handled: '{Handled}', " +
                 "Attempt: '{Attempt}', " +
-                "Execution Time: '{ExecutionTimeMs}'",
```

Comment:
I think this is a breaking change so we shouldn't actually make it.

If users are using the log event parameter names to emit telemetry, this would cause the metrics's name to change and might break things, such as alerts configured based on the metrics' names and values.

Am I correct on my understanding here @martintmk, as I seem to remember this was what you were planning to do in Teams applications with the v8 telemetry. Or have I misunderstood?

Reply:
This `Log` class is marked as `internal`. I don't understand how can this cause a breaking change? Can you please rephrase?

Reply:
The names of the parameters (`ExecutionTimeMs` => `ExecutionTime`) are emitted as part of the structured logging events, which (IIRC) are then consumed to emit metrics.

Reply:
I think changing `ExecutionTimeMs` to `ExecutionTime` is doable. People shouldn't really on log parameter names for anything important. (I hope at least :D)

Reply:
I can revert the renaming of the parameter, but with that the log message pattern will look a bit clumsy and we will continue to have inconsistent parameter naming across two types of log events.

Reply:
It's possible I'm getting confused with this sort of guidance: https://github.com/dotnet/aspnetcore/pull/18494#discussion_r379603655

@martintmk How are you generating your metrics from Polly?

Reply:
For us, the [metrics](https://www.pollydocs.org/advanced/telemetry.html#metrics) are important and effectively fixed, so no breaking changes here. 

For log properties we can be little bit more relaxed. Currently, I don't think anyone relies on this exact name so I am ok with the rename. (old name still works for me though)

Reply:
OK looks like this isn't a breaking change to any metrics, however I think on reflection it's better to keep the unit in the logging property name. Otherwise if looking at the property in a tool (Aspire, Datadog, Kibana...) you're not immediately going to be able to tell the unit as it's just a number. If it were a `TimeSpan` it would be self-explanatory. We can still keep the `ms` in the template for when it's rendered/formatted.

Here's how Polly logging looks in Kibana:

![image](https://github.com/App-vNext/Polly/assets/1439341/57f39562-7e7e-45e8-bc14-4094be2c4631)

With this change, it would change to `ExecutionTime=35.595` and wouldn't be clear if it's milliseconds or seconds (or something else).

Reply:
If we were shooting for self-explanatory field names then `handled`, `attempt`, or `result` are also can be considered as vague field names.

Anyway, I used `ExecutionTimeMs` for `ExecutionAttempt` and `PipelineExecuted` as well.

Reply:
It's less self-explanatory, and more ambiguity. "Attempts" don't have a natural unit, where as things like time, distance, temperature, etc. do, and getting them wrong [can be very expensive](https://en.wikipedia.org/wiki/Mars_Climate_Orbiter#:~:text=An%20investigation%20attributed%20the%20failure%20to%20a%20measurement%20mismatch%20between%20two%20measurement%20systems%3A%20SI%20units%20(metric)%20by%20NASA%20and%20US%20customary%20units%20by%20spacecraft%20builder%20Lockheed%20Martin.%5B3%5D), so it's better to be explicit where the value doesn't inherently convey the units.